helm-charts/common/asr/templates/configmap.yaml

```diff
@@ -0,0 +1,20 @@
+# Copyright (C) 2024 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: {{ include "asr.fullname" . }}-config
+  labels:
+    {{- include "asr.labels" . | nindent 4 }}
+data:
+  {{- if .Values.ASR_ENDPOINT }}
+  ASR_ENDPOINT: {{ .Values.ASR_ENDPOINT | quote}}
+  {{- else }}
+  ASR_ENDPOINT: "http://{{ .Release.Name }}-whisper"
+  {{- end }}
+    #EASYOCR_MODULE_PATH: "/tmp/.EasyOCR"
```

Comment:
if it's not used, should be remove them?

Reply:
will do